### PR TITLE
{vis}[intel-2016a] GL2PS 1.3.9 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
Needs ~~#2624~~ (libXmu)

### CODE REVIEW

**Comment by boegel** [comment]
@hajgato seems like you're missing a `CMake` build dep in the GL2PS easyconfig

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 5 (5 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/aa469c87d92905c138a1 for a full test report.

**Comment by hajgato** [comment]
@boegel corrected (it builds suprisingly fast)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/acc67348a6cac676f868 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c5d0479b67d2a894b3ee for a full test report.

File: easybuild/easyconfigs/f/freeglut/freeglut-3.0.0-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+configopts = ' -DX11_X11_LIB="$EBROOTLIBX11/lib/libX11.so" '
+configopts += ' -DX11_X11_INCLUDE_PATH="$EBROOTLIBX11/include/X11" '
+configopts += ' -DX11_Xext_LIB="$EBROOTLIBXEXT/lib/libXext.so" '
+configopts += ' -DX11_Xrandr_LIB="$EBROOTLIBXRANDR/lib/libXrandr.so" '
+configopts += ' -DX11_Xrandr_INCLUDE_PATH="$EBROOTLIBXRANDR/include/X11/extensions/" '
+configopts += ' -DX11_Xi_LIB="$EBROOTLIBXIlib/libXrandr.so" '
+configopts += ' -DX11_Xi_INCLUDE_PATH="$EBROOTLIBXI/include/X11/extensions/" '
+
+sanity_check_paths = {
+    'files': ['lib64/libglut.a', 'lib64/libglut.%s' % SHLIB_EXT],
```
Comment (boegel) [inline easybuild/easyconfigs/f/freeglut/freeglut-3.0.0-intel-2016a.eb]: this used to be `lib/` before, now it's `lib64`?

should we consider both?

```python
sanity_check_paths = {
    'files': [('lib/libglut.a', 'lib64/libglut.a'), ('lib/libglut.%s' % SHLIB_EXT, 'lib64/libglut.%s' % SHLIB_EXT)],
    'dirs': ['include/GL'],
}
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2668.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/aa6155471d72f6be0c95 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2090.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d1750be5298d51424954 for a full test report.

**Comment by boegel** [comment]
going in, thanks @hajgato!

**Pull request merged**